apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/ProjectTest.cs

```diff
@@ -86,6 +86,38 @@ public void GetAndCheckServiceName()
             Assert.Equal("No Google App Engine service name was passed in or detected.", ex.Message);
         }
 
+        [Fact]
+        public void GetServiceName()
+        {
+            var serviceName = "my-app";
+            var resourceServiceName = "my-app";
```

Comment:
These should not be the same string ;)

Reply:
This was c-n-p, so I'm sure it's the same everywhere - will fix them all.